spec/commands/stories/mark_as_read_spec.rb

```diff
@@ -9,7 +9,7 @@
 
     it "marks a story as read" do
       command = MarkAsRead.new(1, repo)
-      expect(story).to receive(:update_attributes).with(is_read: true)
+      expect(story).to receive(:update).with(is_read: true)
```

Comment:
This is one reason why I generally avoid stubbing methods. Does it work if we change it to something like:

```rb
expect { command.mark_as_read }.to change(story, :is_read).to(true)
```

Reply:
I would prefer that, too. But if you open the file you see that everything is stubbed out. I will change all files to actual model instances that hit the db.